source/_dashboards/energy.markdown

```diff
@@ -204,10 +204,19 @@ max_devices: 5
   Screenshot of the detail devices energy graph card.
 </p>
 
-The **Detail devices energy graph** card is similar to the **Devices energy graph** card, but shows the individual usage on a time scale. 
+The **Detail devices energy graph** card is similar to the **Devices energy graph** card, but shows the individual usage on a time scale.
+
+By default, this card will show all your devices. Optionally, the number of devices can be limited by adding the `max_devices` option and specifying the maximum number of devices to show. If there are more devices available than shown, the devices with the highest energy usage are shown.
```

Comment:
This is just a copy-paste of the description of the same option just above for energy devices graph. Is it better to repeat the entire description verbatim, or just say "this supports the same option as energy-devices-graph" ?

Reply:
good question. I would leave it in, as copy. It leaves less room for ambiguity. I realize there are docs maintenance costs to this.